srml/staking/src/lib.rs

```diff
@@ -585,8 +585,18 @@ decl_storage! {
 					};
 				}
 
+				// Make sure the genesis state of the session module is consistent
+				// with the genesis state of the staking module.
 				if let (_, Some(validators)) = <Module<T>>::select_validators() {
-					<session::Validators<T>>::put(&validators);
+					let keys = validators.iter().map(|validator| {
+						(
+							validator.to_owned(),
+							<session::Module<T>>::next_key_for(validator)
+								.unwrap_or_default()
```

Comment:
```suggestion
```

Reply:
While lines up to 120 are allowed, I still think it's a good practice to stick to 80 if possible, so that you can have to files open side by side without line wrapping.